tests/common/fixtures/advanced_reboot.py

```diff
@@ -168,7 +168,17 @@ def __buildTestbedData(self, tbinfo):
         self.vlanMaxCnt = len(self.mgFacts['minigraph_vlans'].values()[0]['members']) - 1
 
         self.rebootData['dut_hostname'] = self.mgFacts['minigraph_mgmt_interface']['addr']
+        config_facts = self.duthost.get_running_config_facts()
+        vlan_table = config_facts['VLAN']
+        vlan_name = list(vlan_table.keys())[0]
+        vlan_mac = vlan_table[vlan_name]['mac']
+        self.rebootData['vlan_mac'] = vlan_mac
         self.rebootData['dut_mac'] = self.duthost.facts['router_mac']
+        if 'dualtor' in tbinfo['topo']['name']:
+            self.rebootData['lo_prefix'] = "10.1.0.33/32"
```

Comment:
Can we get this data from duthost.facts or config facts, instead of hardcoding?

Reply:
Updated